Stripe/STPCard.m

```diff
@@ -191,17 +192,23 @@ - (NSString *)stripeID {
     return self.cardId;
 }
 
-- (NSString *)label {
+- (NSString *)paymentMethodLabel {
     NSString *brand = [self.class stringFromBrand:self.brand];
     return [NSString stringWithFormat:@"%@ %@", brand, self.last4];
 }
 
-- (UIImage *)image {
+- (UIImage *)paymentMethodImage {
     return [STPImageLibrary brandImageForCardBrand:self.brand];
 }
 
-- (UIImage *)templateImage {
+- (UIImage *)paymentMethodTemplateImage {
     return [STPImageLibrary templatedBrandImageForCardBrand:self.brand];
 }
 
+- (STPPaymentMethodType *)paymentMethodType {
+    // TODO: Should this be applePay for apple pay created cards?
```

Comment:
+1 for checking `isApplePayCard` and making this `applePay` if it's an apple pay token